.github/workflows/version.yaml

```diff
@@ -0,0 +1,47 @@
+name: Version
+on:
+  workflow_dispatch:
+    inputs:
+      packages:
+        description: 'Selected packages as comma separated string, e.g. modules/storage-spec,libraries/formatting'
+        type: string
+        required: true
+      version-strategy:
+        type: choice
+        options:
+          - conventional-commits
+          - patch
+          - minor
+          - major
+          - prerelease
+          - prepatch
+          - preminor
+          - premajor
+        default: 'conventional-commits'
+      assignee:
+        description: 'Overrides the default assignee, which is the user to trigger this workflow.'
+        type: string
+        required: false
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: ExodusMovement/actions/setup/lerna@06f077a9081a1e973586dde8870e7116873b9b84
```

Comment:
@sparten11740 should we use the latest here?

Reply:
yes